roles/etcd/vars/main.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 # renovate: datasource=github-releases depName=etcd-io/etcd
-etcd_version: 3.5.9
+etcd_version: v3.5.10
```

Comment:
Need to update usages to handle this version string, or update renovate rule to strip.